src/service.ts

```diff
@@ -262,6 +271,12 @@ export class DebugService implements IDebugger {
     this._model.variablesModel.scopes = [];
   }
 
+  private clearModel() {
+    this.onContinued();
```

Comment:
This `onContinued` feels a bit off in the `clearModel` method.

Maybe we can move its content to `clearModel`? (or rename it)

Reply:
We cannot move it inside `clearModel` since `onContinue` is not supposed to clear the breakpoints.

Reply:
:+1: